### DISCUSSION TITLE
Who's hiring (Dec 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#43787] [Ideas]
# What is this?
New month, new post! This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #43786.

*If you're looking for last month's discussion, see #42311. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!*

**Comment by stevo2105** [comment]
Whop ([whop.com](https://whop.com) / [dash.whop.com](https://dash.whop.com)) | Senior Frontend Eng | Brooklyn NY | Full-time | $170-230k base | [Apply here](https://jobs.polymer.co/whop/26288)

**Reply by kmoussai** [reply]
Hey @stevo2105 , I'm interested in this position please review my application.

**Comment by AlessandroVol23** [comment]
Hashnode | Fully REMOTE & Async | Full-Time

Multiple roles (Engineering, Design, Product)

Hashnode is the '#1 blogging platform for developers that allows you to publish articles on your own domain and helps you stay connected with a global developer community. You own your content!

The main tech role is:

Senior Fullstack Developer: Develop the features for the next-gen developer platform. -> https://careers.hashnode.com/jobs/lg_F08w6be30/software-developer-full-stack-javascript-remote

The tech-stack:

Fully deployed on Vercel
Next.js, React, Tailwind
TypeScript
MongoDB
GraphQL
AWS Serverless - CDK & SST - Eventbridge Lambda, SQS, DynamoDB, Api Gateway

We love an open writing culture & transparency so you can share with the world what you are working on.
Hashnode is hiring remotely all over the globe (not just US only). 
We work async, which means you set your own schedule working.
We also have a growing [engineering blog](https://engineering.hashnode.com/) if you're interested in some topics in more detail.

Hashnode is backed by many great VCs (e.g. Salesforce Ventures, Accel Partners, Sierra Ventures, Sequoia Capital's Surge) and individual investors like Vercel's CEO Guillermo Rauch, AngelList's founder Naval Ravikant, Intercom's founder Des Traynor, Ed Roman, and Salil Deshpande.

Contact me: at [sandro@hashnode.com](mailto:sandro@hashnode.com) / twitter.com/sandro_vol for more info or check out the careers page: https://careers.hashnode.com/jobs

Contact me for any questions I am happy to help :)

**Reply by mocherfaoui** [reply]
I remember I applied for a job offer at hashnode nearly three months ago and didn't even receive a rejection letter...

**Comment by riethmayer** [comment]
Automata ([automata.tech](https://automata.tech)) [Fullstack Typescript Engineer](https://careers.automata.tech/jobs/2281166-senior-full-stack-engineer-typescript) | London, UK | Full-time | GBP 65-95k base | Remote | [Apply Here](https://careers.automata.tech/jobs/2281166-senior-full-stack-engineer-typescript)

Automata is building a world-class company to disrupt the life science industry and empower scientists by making automation a reality for every lab.

We work with labs across the world to understand their specific workflows and needs, and develop personalised solutions that range from diagnostic, drug discovery to synthetic biology. This allows us to simplify common automation tasks, while also supporting more advanced paths for specialised workflows, in a fast, flexible, and cost-efficient way.

We are looking for a Senior Fullstack Engineer - Typescript whose mission is to be helping technicians make a transition from manual and repetitive workers to knowledge workers, operating the next generation of smart bench automation.

## Why is this role important for Automata? 

For automation to take over, scientists need to master operating multi-million dollar benches and feel that Automata has their back to avoid mistakes.

**In this role, you will be responsible for:** 

- Developing applications that focus on technicians' day-to-day operation of Automata's product
- You'll own the product end-to-end, creating a world-leading run experience for smart benches

**What is takes:**

- 3 years of commercial experience with Typescript and advanced Typescript definitions
- 3 years of commercial experience with React
- Experience with Prisma
- Experience with Nextjs
-  Experience with real-time applications (WebRTC, Pusher, Ably)

**Nice to have:** 

- Auth0
- tRPC
- React Native
- Electron

**What gets us excited?** 

We’re passionate about the future of Technology and Automation and our growing success is built upon like-minded individuals who share our passion. 

You should apply if you're excited about delivering great experiences to customers using cutting-edge technology while setting a high bar on lighthouse scores.

Salary band:

£65,000 -£95,000 depending on experience + bonus and equity

**Comment by dhruv92** [comment]
Create | Senior Full Stack Developers | Remote | Freelance | $50-$75 per hour 

Create (https://create.xyz) aims to speed up development through code generators for common parts of modern web applications. We're looking for a few more experienced full stack dev freelancers to join the team, either full time or part time. We also have roles open for frontend and backend engineers. 

The products we create are web apps built on the following stack: 

Typescript
NextJS
TailwindCSS
Radix UI
Apollo
Node
GraphQL
TypeORM
Postgres

Please apply at https://create.xyz/apply. Looking forward to meeting you :)

**Reply by felipe-muner** [reply]
Hello people,

If you are looking for motivated folks and aiming to Create big stuff this is the place to go! I've been working with @dhruv92 and only have good stuff to say about all the onboarding process and so on =).

Lets go 🚀

**Comment by sivarcot** [comment]
Deestree |Full stack developer | EU Zone preferred | Freelance convertible to fulltime | Dependent on location and XP | https://www.linkedin.com/hiring/jobs/3407402092/detail/

We are looking for top-notch full stack developers willing to change how food is sold in the B2B space. Remote freelance based in EU zone preferred, possible permanent position.

If you are passionate about technologies and you are a fast learner, you are the right person to join Deestree! Please contact: job-tech@deestree.com


**The company**

Deestree.com is a new wholesale food B2B marketplace built for independent suppliers to gain visibility. Suppliers are at last free to sell more easily at the price they want.

We are an early stage tech startup, growing super fast. We are onboarding our first customers and investors. We are building from scratch a new platform using the latest technologies and best development practices. We are obsessed with providing the best user experiences and simplicity to our users.

Deestree was founded by 3 domain experts: Ex-Amazon tech leader, multi startups founder, wholesale business owner.


**Your mission**

We will be working as freelance, optionally willing to move to a permanent position as a tech lead.

- Build features ground to top from the infrastructure (AWS, serverless, python, terraform etc) up to the UI (NextJS, Typescript, Bootstrap etc).
- Bring your experience to deliver a bug free platform to minimize customer impact.

**Your background**

We are not looking for language experts, we are looking for engineers who can solve problems.

- Engineering degree or equivalent
- 5+ years of experience as a software engineer
- An experience in front end development
- Strong experience in building scalable products
- Fluent in English

If you want to have more details, feel free to contact us: contact@deestree.com

**Reply by Babbili** [reply]
i got Mail Delivery failed on your email job-tech@deestree.com

**Reply by sivarcot** [reply]
> i got Mail Delivery failed on your email [job-tech@deestree.com](mailto:job-tech@deestree.com)

Thanks a lot for letting me know. The problem is now solved.

**Comment by johnsononextech** [comment]
One X Tech | Full-Stack Software Engineer | Remote (Europe,  Asia) | Contract | $25-35 hourly rate | 

We are a software development company that focuses on building technological innovations for leading enterprises.
 
We are looking for a highly motivated Software Engineer with 3 years of Javascript experience to join our team of passionate engineers. If you are not afraid of challenges, eager to learn or use the latest technologies, build strong web applications, work, share and learn with a team of tech experts, then we are looking forward to your application.

## Key responsibilities
• Build and maintain scalable and high performing web sites and solutions, consistent with our quality guidelines.
• Design, and implement new sections/features/content for new and existing websites.
• Develop and optimise existing features, as well as prototype and implement new projects.
• Ensure that Front-end websites are responsive, cross-browser and on mobile.
• Work with UI/UX design team to bring digital products to life

## Technologies
• ReactJS, Javascript, and TypeScript experience
• High degree of fluency in JavaScript, with knowledge of modern techniques as well as core JS concepts
• Proficiency in frontend markup and styling - HTML and CSS.
• Able to design cross-browser and responsive pixel-perfect HTML. Be familiar with Material-UI.

## Requirements
• Min. 3 years experience with relevant technologies above.

Interested applicants can send their resumes and portfolio to careers@onextech.com

**Comment by leif-cycle** [comment]
cycle ([cycle.de](https://cycle.de)) | OG of FrontEnd | remote or Hamburg/Frankfurt | ≥20 h/week | equity & 1k per Month to live life | 

We are looking for a _NextJS Master of Desaster_ to revolutionize the bike trading market at [cycle.de](https://cycle.de). Join us and ramp up [cycle.de](https://cycle.de) for the 2023 season.

**Teamwork & Techstack:** As a team we connect through Discord, Postman, Figma & Notion. At the FrontEnd are working with NextJS/vercel (well, captain obvious strikes again). The BackEnd API is well documented and has a performant & clean structure.

We build and bootstrap cycle.de & admin.cycle.de, the (Mail-)Server, CMS, bots, analytics and all other fancy stuff to be bloody good.

**YOU:** Besides being a little cool & crazy, you should be capable of coding strong and clean NextJS. Age, sex & origin doesn't matter, experience and proof of your work experience does.

HMU and we'll find out if we are a match. Just write us at ezclap@cycle.de. I'll reply asap, though prob in 2023.

Ahu!

**Comment by Recruitment-VMS** [comment]
Looking to hire an **on-site** **Full Stack Engineer** at Noida, India 
Company Name: **VMS Techs**

**Job description**
We are looking for a skilled Full Stack Engineer responsible for developing and/or designing.
To succeed in this role, you will need to be proficient in Laravel, CI, PHP, JavaScript, HTML, CSS, and have solid knowledge and experience in programming applications.

**Responsibilities**
•	Work with development teams and product managers to ideate software solutions
•	Design client-side and server-side architecture
•	Build the front-end of applications through appealing visual design
•	Develop and manage well-functioning databases and applications
•	Write effective APIs
•	Test software to ensure responsiveness and efficiency
•	Troubleshoot, debug and upgrade software
•	Create security and data protection settings
•	Build features and applications with a mobile responsive design
•	Write technical documentation
•	Work with data scientists and analysts to improve software

**Requirements and skills**
•	Degree in Computer Science, Statistics or any relevant field
•	Experience developing desktop and mobile applications
•	Familiarity with common stacks
•	Knowledge of multiple front-end languages and libraries (e.g. HTML/ CSS, JavaScript, XML, jQuery)
•	Knowledge of multiple back-end languages (e.g. C#, Java, Python) and JavaScript frameworks (e.g. Angular, React, Node.js)
•	Familiarity with databases (e.g. MySQL, MongoDB), web servers (e.g. Apache) and UI/UX design
•	Excellent communication and teamwork skills
•	Proven experience as a Full Stack Developer or similar role
•	Must be able to pay attention to detail
•	Excellent Organizational skills
•	Analytical skills

Interested Candidate with **3+ years of experience** may reach out at **recruitment@vmstechs.com**
Would really appreciate it, if anyone can recommend or share the post in their circle of developers. 
Thanks 
Harpreet Kaur
